notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,2260 @@
+{
```

Comment:
<p>Space-time could be a cool example! We have "minkowski metric" implemented in geomstats btw. </p><p> </p><p>https://en.wikipedia.org/wiki/Minkowski_space</p><p> </p><p>https://github.com/geomstats/geomstats/blob/62a378d2a0f87cb6942a3a0e91e289b09bef9396/geomstats/geometry/minkowski.py</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Good idea! I added this</p>